bark/io/datfromwav.py

```diff
@@ -0,0 +1,36 @@
+from scipy.io import wavfile
+import bark, os
+
+
+def dat_from_wav(wav, barkname, **attrs):
+    rate, data = wavfile.read(wav)
+    return bark.write_sampled(barkname, data, rate,**attrs)
+
+
+def _main():
+    ''' Function for getting commandline args.'''
+    import argparse
+
+    p = argparse.ArgumentParser(description='''
+    converts wav file to bark format
+        ''')
+    p.add_argument('wav', help='path to wav file')
+    p.add_argument('out', help="path to bark file")
+    p.add_argument("-a",
+        "--attributes",
+        action='append',
+        type=lambda kv: kv.split("="),
+        dest='keyvalues',
+        help="extra metadata in the form of KEY=VALUE")
+
+    args = p.parse_args()
+    if args.keyvalues:
```

Comment:
I don't think you need this conditional - if `args.keyvalues` is empty, `**dict(args.keyvalues)` should work just as you'd expect.

Reply:
We need this additional check to determine our args.keyvalues is not an instance of None. B/c by default args.keyvalues is set to 'None' and not an empty dict.
If we skip this check then it throws an error complaining 'Nonetype' object is not iterable if we try to make dict out of this None object.

Reply:
A better solution to this problem, I think, is to specify a `default` value in the appropriate `add_argument` call - `default=[]` should work.

Then you can get rid of the check.